lighthouse-core/test/audits/seo/font-size-test.js

```diff
@@ -14,10 +14,12 @@ const validViewport = 'width=device-width';
 /* eslint-env jest */
 
 describe('SEO: Font size audit', () => {
+  const makeMetaElements = viewport => viewport ? [{name: 'viewport', content: viewport}] : [];
+
   it('fails when viewport is not set', () => {
     const artifacts = {
       URL,
-      Viewport: null,
+      MetaElements: makeMetaElements(null),
```

Comment:
Is the conditional in `makeMetaElements` just for this case? `null` could just be `undefined` or this one could just be `[]` an no conditional needed :)